easybuild/easyblocks/s/superlu.py

```diff
@@ -44,7 +47,7 @@ class EB_SuperLU(CMakeMake):
     @staticmethod
     def extra_options():
         extra_vars = CMakeMake.extra_options()
-        extra_vars['build_shared_libs'][0] = False
+        extra_vars['build_shared_libs'][0] = True
```

Comment:
I think I would just drop this change for now, it shouldn't hold back the other updates (which are necessary not nice-to-haves). If you feel strongly about it open a new PR.

Reply:
I traced back the introduction of this option (to #860), and there seem's to be no particular reason for this default. You could keep previous behaviour by ensuring that static _and_ shared libraries are built.

Reply:
I have changed that back to False. The more important thing is to get FelxiBlas working.